### PR TITLE
Iris constants definition

### PR DESCRIPTION
**Pull request by marcos-iov**
Missing 

- btcHeightWhenBlockIndexActivates (BTC height when Iris activates in RSK)

*fed:3.0.0.0*
*fit:fix-flyove-test*
*ci:iris-update*

### CODE REVIEW

File: rskj-core/src/main/java/co/rsk/config/BridgeDevNetConstants.java [hunk 1]
```diff
@@ -151,8 +150,7 @@ public BridgeDevNetConstants(List<BtcECKey> federationPublicKeys) {
         // e1b17fcd0ef1942465eee61b20561b16750191143d365e71de08b33dd84a9788
         oldFederationAddress = "2N7ZgQyhFKm17RbaLqygYbS7KLrQfapyZzu";
 
-        minSecondsBetweenCallsReceiveHeader = 600;  // 10 minutes in Seconds
-
+        minSecondsBetweenCallsReceiveHeader = 600;  // 10 minutes in seconds
```
Comment (donequis) [inline rskj-core/src/main/java/co/rsk/config/BridgeDevNetConstants.java:153]: This value seems too big

File: rskj-core/src/main/java/co/rsk/config/BridgeDevNetConstants.java [hunk 2]
```diff
@@ -130,10 +130,9 @@ public BridgeDevNetConstants(List<BtcECKey> federationPublicKeys) {
         lockingCapIncrementsMultiplier = 2;
 
         btcHeightWhenBlockIndexActivates = 700_000; //TODO define this value when Iris activation height in RSK is determined
+        maxDepthToSearchBlocksBelowIndexActivation = 4320; // 30 days in BTC blocks (considering 1 block every 10 minutes)
 
-        maxDepthToSearchBlocksBelowIndexActivation = 1_000; //TODO define this value with Sergio
-
-        erpFedActivationDelay = 1000;
+        erpFedActivationDelay = 5260; // 1 year in BTC blocks (considering 1 block every 10 minutes)
```
Comment (donequis) [inline rskj-core/src/main/java/co/rsk/config/BridgeDevNetConstants.java:135]: 52560

File: rskj-core/src/main/java/co/rsk/config/BridgeMainNetConstants.java [hunk 3]
```diff
@@ -129,17 +129,16 @@
         initialLockingCap = Coin.COIN.multiply(300); // 300 BTC
 
         btcHeightWhenBlockIndexActivates = 700_000; //TODO define this value when Iris activation height in RSK is determined
-        maxDepthToSearchBlocksBelowIndexActivation = 1_000; //TODO define this value with Sergio
+        maxDepthToSearchBlocksBelowIndexActivation = 4320; // 30 days in BTC blocks (considering 1 block every 10 minutes)
 
-        // TODO: This value is not real
-        erpFedActivationDelay = 5063;
+        erpFedActivationDelay = 5260; // 1 year in BTC blocks (considering 1 block every 10 minutes)
```
Comment (donequis) [inline rskj-core/src/main/java/co/rsk/config/BridgeMainNetConstants.java:134]: 52560

Comment (marcos-iov) [reply rskj-core/src/main/java/co/rsk/config/BridgeMainNetConstants.java:134]: Good catch, thanks!

**Review by josedahlquist — APPROVED** [review]

**Pull request merged**